main_test.go

```diff
@@ -49,3 +49,20 @@ func TestFindRoot(t *testing.T) {
 		}
 	}
 }
+
+func TestSlashedGOPATH(t *testing.T) {
+	gopath := os.Getenv("GOPATH")
```

Comment:
oh yeah we might want to use our fake go path helpers, but i dont know if they will be able to work to test as well :/